lib/dest/write-contents/write-symbolic-link.js

```diff
@@ -2,19 +2,17 @@
 
 var fs = require('graceful-fs');
 
+var fo = require('../../file-operations');
+
 function writeSymbolicLink(file, onWritten) {
   // TODO handle symlinks properly
   fs.symlink(file.symlink, file.path, function(symlinkErr) {
-    if (isFatalError(symlinkErr)) {
+    if (fo.isOverwriteError(symlinkErr)) {
```

Comment:
I think this is the inverse of naming/usage.  I believe it's **not** an overwrite error if it returns `true`.  Maybe I'm wrong?